From 017fc2da5681a18c5e379c11c5d55c82d49da079 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@apache.org>
Date: Fri, 11 Aug 2017 11:41:16 -0700
Subject: [PATCH 2607/2848] HADOOP-14760. Add missing override to
 LoadBalancingKMSClientProvider.

(cherry picked from commit 07fff43f4a1e724c83ff8fcc90fac64aa04a39eb)

 Conflicts:
	hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java

Change-Id: I6fd731607acbcb4ae51b35cd8e8d100fd6a3747d
---
 .../key/kms/LoadBalancingKMSClientProvider.java    |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
index 8205862..e0e4c7f 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/LoadBalancingKMSClientProvider.java
@@ -294,6 +294,7 @@ public KeyVersion call(KMSClientProvider provider)
     }
   }
 
+  @Override
   public EncryptedKeyVersion reencryptEncryptedKey(
       final EncryptedKeyVersion ekv)
       throws IOException, GeneralSecurityException {
-- 
1.7.9.5

